src/Login/Login.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import {BrowserRouter, Route, Routes } from "react-router-dom";
+import "./Login.css";
+
```

Comment:
The 'Login' subfolder should be under 'Pages'

Pages -> Login -> {login.tsx/login.css}